### PR TITLE
Fix Audio player should be fixed to bottom (#135)

### PR DESCRIPTION
**Pull request by iamonuwa**
1. Fix audio player to bottom

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -128,18 +128,22 @@ export default {
...
 
     .toast {
       left: 19rem;
     }
 
     > aside {
       width: 19rem;
-      position: sticky;
+      position: fixed;
+      z-index: 1;
```
[the remark this answers is not in the document]
Comment (iamonuwa) [reply src/App.vue]: Fixed

File: src/App.vue [hunk 2]
```diff
@@ -128,6 +128,7 @@ export default {
     justify-content: flex-start;
     align-items: flex-start;
     position: relative;
+    overflow-x: hidden;
```
Comment (chovy) [inline src/App.vue]: we shouldn't have to hide here on the x-axis. there is something else causing the problem.

Comment (iamonuwa) [reply src/App.vue]: Looking at it again

File: src/store/modules/posts.js [hunk 3]
```diff
@@ -8,6 +8,7 @@ const actions = {
     }),
   [types.POST_LIST]: ({ commit }) => posts.getAll()
     .then((res) => {
+      console.log(res.data);
       commit(types.POST_LIST, res.data.results);
```
Comment (chovy) [inline src/store/modules/posts.js]: remove console.log

**Pull request merged**